examples/basic/README.md

```diff
@@ -18,4 +18,6 @@ npm run bs4:dev
 npm run bs3:prod
 # Bootstrap 4
 npm run bs4:prod
+
+### See .bootstraprc for configuration details
```

Comment:
This is changing. See #114.

/examples/basic
.bootstraprc-3-example
.bootstraprc-4-example

and 

/examples/css-modules
.bootstraprc-3-example
.bootstraprc-4-example